### PR TITLE
[IOTDB-3558] Dump node stacks if IT test request timeout

### PR DESCRIPTION
**Pull request by ericpai**
JIRA: https://issues.apache.org/jira/browse/IOTDB-3558
For any test case failed by request timeout, a thread dump file will be output to `cluster-logs` dir of every node.
image

### CODE REVIEW

File: integration-test/src/main/java/org/apache/iotdb/it/env/AbstractEnv.java [hunk 1]
```diff
@@ -200,14 +193,53 @@ public void cleanAfterTest() {
...
+              .collect(Collectors.toList());
+      String cmd = getSearchAvailablePortCmd(requiredPorts);
+
+      try {
+        Process proc = Runtime.getRuntime().exec(cmd);
+        if (proc.waitFor() == 1 && lockFile.createNewFile()) {
+          return requiredPorts.stream().mapToInt(Integer::intValue).toArray();
+        }
+      } catch (IOException | InterruptedException ex) {
+        // ignore
```
Comment (wangchao316) [inline integration-test/src/main/java/org/apache/iotdb/it/env/AbstractEnv.java:217]: Is  add "Thread.currentThread().interrupt();"   ?

**Review by wangchao316 — COMMENTED** [review]
Good

File: integration-test/src/main/java/org/apache/iotdb/it/env/AbstractEnv.java (hunk 1 above)
Comment (ericpai) [reply integration-test/src/main/java/org/apache/iotdb/it/env/AbstractEnv.java:217]: Good catch! It has been fixed.

**Review by irvine0109 — APPROVED** [review]
LGTM

**Pull request merged**